### ISSUE TITLE
Lightshouse Accessibility Test Fails due to bug in lighthouse

### ISSUE THREAD

**Issue by simonmaass** [DivanteLtd/vue-storefront#4555]
## What is the motivation for adding / enhancing this feature?

in the current 1.12 branch the lighthouse test for accessibility has a problem with the viewport attribute in relation to zooming:

![image](https://user-images.githubusercontent.com/32599710/84884087-9385fe00-b091-11ea-8681-9b1d291db26f.png)

## What are the acceptance criteria 

maximum-scale should be set to 5
- [ ] Run lighthouse test on demo site and 100% accessibilty score

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [x] This is a normal feature request. This should be available on https://test.storefrontcloud.io and then after tests this can be added to next Vue Storefront version. In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] (**Pick this option only if you're sure**) This is an important improvement request for current Release Candidate version on https://next.storefrontcloud.io and should be placed in next RC version. In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] (**Pick this option only if you're sure**) This is a critical improvement request for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version. In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

**Comment by simonmaass** [comment]
actually i think its the bug in the meta package: https://github.com/dequelabs/axe-core/pull/2238

**Comment by pkarw** [comment]
@simonmaass can you create a PR with a fix please?

**Comment by simonmaass** [comment]
Marking this as closed since 6.1 has shipped with these fixes in https://github.com/GoogleChrome/lighthouse/pull/10986

as soon as 6.1 is release on web.dev the accessibility score should work again
